### ISSUE TITLE
settings.where not working?

### ISSUE THREAD

**Issue by ljack** [WillBrock/volcano-table#1]
I've this:

``` javascript
Template.columnsMapping.helpers({
    example_settings: function() {
        console.log("volcano-table example_settings");
        return {
            collection: Infos,
            where: { createdBy: Session.get("selected_user") } ,
```

But it seems to me that the **_where**_ is having no effect, everything in Infos is shown in the table instead. 

I's browsing the source code and couldn't figure out where the **_settings.where**_ is actually taken into account?

 I know that I'm probably doing something wrong, just can't figure out what it is...

Created a patch https://github.com/WillBrock/volcano-table/pull/2

**Comment by ghost** [comment]
Hi ljack,

Did you manage to make this work?
I'm having the same problem.
(I want to show only the saleslines for a selected salesorder)

**Comment by ljack** [comment]
Yes, I actually created a local version of this package and have since modified it heavily.

**Comment by ghost** [comment]
Are you willing to share that code?

**Comment by ljack** [comment]
I think this https://github.com/ljack/volcano-table fork contains that pacth#2 mentioned in the first post.

**Comment by ghost** [comment]
Thanks! I'll have a look at it
